server/lomas_server/dp_queries/dp_libraries/opendp.py

```diff
@@ -17,6 +19,88 @@
 from utils.loggr import LOG
 
 
+def get_lf_domain(metadata):
+    """
+    Returns the OpenDP LazyFrame domain given a metadata dictionary.
+
+    Args:
+        metadata (dict): The metadata dictionary
+
+    Raises:
+        Exception: If there is missing information in the metadata.
+
+    Returns:
+        dp.mod.Domain: The OpenDP domain for the metadata.
+    """
+    series_domains = []
+
+    # Series domains
+    for name, series_info in metadata["columns"].items():
+        if series_info["type"] in ["float", "int"]:
+            series_type = f"{series_info['type']}{series_info['precision']}"
+        else:
+            series_type = series_info["type"]
+
+        # TODO should this be a constant? leave here
+        opendp_type_mapping = {
```

Comment:
yes, I think it can be a constant

Reply:
Let's clear that with the ticket I created for this.